docs/index.rst

```diff
@@ -168,6 +168,13 @@ This will produce the following plot, which visualizes the electromagnetic field
 
 You can now postprocess simulation data using the same python session, or view the results of this simulation on our web-based `graphical user interface (GUI) <https://tidy3d.simulation.cloud>`_.
 
+.. tip::
+
+   Repeated runs of the same simulation can reuse solver results by enabling the optional
+   local cache: ``td.config.local_cache.enabled = True``. The cache location and limits are
+   configurable (see ``~/.tidy3d/config``), entries are checksum-validated, and you can clear
+   all stored artifacts with ``tidy3d.web.cache.clear()``.
```

Comment:
It still seems to me like a detailed explanation about usage and interaction with the server cache is missing.

For example even I currently don't understand some details, like what happens if I exceed the max entries or the max memory limit. Looking at the tests, it seems like it's first-in first-out as one would expect, but we should make this more explicit either here or in the corresponding Config fields.